frontend/public/components/overview/index.tsx

```diff
@@ -764,30 +781,62 @@ class OverviewMainContent_ extends React.Component<OverviewMainContentProps, Ove
     const {statefulSets} = this.props;
     return _.map(statefulSets.data, (ss) => {
       const buildConfigs = this.getBuildConfigsForResource(ss);
-      const obj = {
-        ...ss,
-        kind: StatefulSetModel.kind,
-      };
-      const readiness = {
-        desired: ss.spec.replicas || 0,
-        ready: ss.status.replicas || 0,
-      };
       const pods = this.getPodsForResource(ss);
       const alerts = combinePodAlerts(pods);
       const services = this.getServicesForResource(ss);
       const routes = this.getRoutesForServices(services);
+      const obj = {
+        ...ss,
+        kind: StatefulSetModel.kind,
+      };
+      const status = <OverviewItemReadiness
+        desired={ss.spec.replicas}
+        href={`${resourceObjPath(obj, obj.kind)}/pods`}
+        ready={ss.status.replicas}
+      />;
+
       return {
         alerts,
         buildConfigs,
         obj,
         pods,
-        readiness,
         routes,
         services,
+        status,
       };
     });
   }
 
+  createPodItems(): OverviewItem[] {
+    const {pods} = this.props;
+    return _.reduce(pods.data, (acc, pod) => {
+      const owners = _.get(pod, 'metadata.ownerReferences');
+      const phase = _.get(pod, 'status.phase');
+      if (!_.isEmpty(owners) || phase === 'Succeeded') {
```

Comment:
I'm worried you might have some Failed pods that don't get deleted, which you won't be able to hide. (This can happen with evicted pods for instance.) I'd hide Failed, too.

```suggestion
      if (!_.isEmpty(owners) || phase === 'Succeeded' || phase === 'Failed') {
```